go.mod

```diff
@@ -0,0 +1,45 @@
+module github.com/nokia/danm
+
+go 1.13
+
+require (
+	github.com/apparentlymart/go-cidr v1.0.1
+	github.com/containernetworking/cni v0.7.1
+	github.com/containernetworking/plugins v0.8.5
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/intel/multus-cni v0.0.0-20200316130803-079c853eba60
+	github.com/intel/sriov-cni v2.1.0+incompatible
+	github.com/j-keck/arping v1.0.0
+	github.com/satori/go.uuid v1.2.1-0.20181028125025-b2ce2384e17b
+	github.com/vishvananda/netlink v1.1.1-0.20200221165523-c79a4b7b4066
+	k8s.io/api v0.17.4
+	k8s.io/apimachinery v0.17.4
+	k8s.io/client-go v0.17.4
+	k8s.io/code-generator v0.17.4
+	k8s.io/kubernetes v1.14.10
```

Comment:
you dont need this anymore I guess with the hacks?

Reply:
`go mod tidy` generated those lines, it wasn't me! 😄 It comes back even if I delete them.

Reply:
ah okay, got it :)